src/map/map.h

```diff
@@ -439,6 +439,16 @@ enum elements {
 	ELE_ALL = 0xFF
 };
 
+/**
+ * Types of Ball Types
+ * Used by clif_spiritball [KeiKun]
+ */
+enum spirit_ball_types {
+	BALL_TYPE_NONE = 0,
+	BALL_TYPE_SPIRIT,
+	BALL_TYPE_SOUL
```

Comment:
you sure it using names like this?

Reply:
I just used those to avoid using `int` as type. is there an official names for it?

Reply:
i cant find real enum, but is this can support ball type bigger than 2? is client draw it in some way?